MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -248,7 +249,7 @@ private static IEnumerable<Directory> ProcessChunk(PngChunk chunk)
             else if (chunkType == PngChunkType.iTXt)
             {
                 var reader = new SequentialByteArrayReader(bytes);
-                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(_latin1Encoding);
+                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(_utf8Encoding);
```

Comment:
There's actually now an issue slightly below here. The `bytesLeft` value was based on the length of the string in bytes, which for latin1 is the same as the length of the string in characters. With UTF-8 that's not the case. I'll patch this up and push to your PR.